### DISCUSSION TITLE
Component not found in CI

### DISCUSSION THREAD

**Discussion by KGKallasmaa** [vercel/next.js#19435] [Help] [answered]
# Bug report

Application builds and behaves normally on local machine, but it fails on Github CI.

## Describe the bug

I have two components located in, footer.tsx and home.tsx. I want to use the footer component in the home page component. It can be done on my development machine, but not in CI.

I didn't experience this behavior when vanilla React was used to build the application. 

## Expected behavior

I expect the build to fail locally.

## Screenshots

![Screen Shot 2020-11-23 at 11 18 40](https://user-images.githubusercontent.com/29797627/99945681-b83b7b80-2d7d-11eb-8fe2-25ab96f3c726.png)

## System information

Development:
- Version of Next.js: [e.g. 10.0.2]
- Version of React: [e.g. 17.0.1]
- Version of Typescript: [e.g. 4.1.2]

**Comment by jamesmosier** [comment, marked as answer]
This usually happens because Linux environments (which most CI are run on) are case sensitive while Mac OS is not.

Therefore this could be 1 of 2 things:

1. Your file or folder contains an uppercase letter in the name but your import path does not
2. Your file or folder at one time contained an uppercase letter, you committed it to Git and later changed the case of it but Git does not recognize this change. Therefore it is best to delete this folder and file, commit it to Git, and then re-add it with the correct casing.
